industry_benchmarks/utils/fix_networks.py

```diff
@@ -59,22 +60,43 @@ def _get_check_results_json(filename: str):
 
 
 def get_transformation_alternate(
-    pur: gufe.ProtocolUnitResult, ligand_network: LigandNetwork
-):
+    pur: dict, alchemical_network: AlchemicalNetwork
+) -> tuple[Transformation, str]:
     """
     Getting a transformation if things got deleted.
+
+    We do this by looking up the gufe-key of the inputs stored in the unit result
     """
-    # The way this was being done in PR #144 doesn't work because
-    # it would lead to either a solvent or vacuum simulation, not a
-    # complex one!
-    errmsg = ("The transformation data is not available, "
-              "please contact the OpenFE team")
-    raise ValueError(errmsg)
+    # grab the gufe key of the chemical systems used in the inputs
+    unit_result = list(pur["unit_results"].values())[0]
+    state_a_key = unit_result["inputs"]["stateA"][":gufe-key:"]
```

Comment:
I thought we had del-ed the entire dictionary entry for these? https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/blob/e376e39445118058e1904f86703a70ac1d468822/industry_benchmarks/utils/results_cleanup.py#L385

Reply:
I think the script removes the components in the `protocol_result` but leaves the key of the system in the `unit_results` which are stored in a different place. I generated the `bace_results/cleaned_results/complex_spiro10_spiro6.json` using the script in that commit to check it works but might have missed something. Do you have a copy of a cleaned directory I can test this on?

Reply:
Ah yes, you're right! please ignore me